SOP/SOP-operator.adoc

```diff
@@ -0,0 +1,48 @@
+ifdef::env-github[]
+:status:
+:tip-caption: :bulb:
+:note-caption: :information_source:
```

Comment:
Missing:

- Prometheus Alerts Procedures
- Validate
- Monitoring

See here: https://github.com/aerogear/unifiedpush-operator/blob/master/SOP/SOP-operator.adoc#prometheus-alerts-procedures

Reply:
About the Prometheus Alerts Procedures, it seems to me that that is duplicating the Overview section. It's pretty much the same sentence and set of links so perhaps we could include one rather than both.

I renamed the `Validate` section to `Check Operator` so it's still in there just under a different name, the link to it references `Check Operator` rather than validate so it does work. 

The Monitoring section in ups includes the following text
```
== Monitoring

If the https://github.com/aerogear/unifiedpush-operator#monitoring-service-metrics[Monitoring Service (Metrics)] is enabled for the installation, a Grafana Dashboard titled `UnifiedPush Operator`, and the Prometheus Monitoring instance are created.
```
so I will include similar text for that here.

Reply:
Actually I'm wondering now if we need to include this text at all. If the user has landed on the SOP then probably the monitoring stuff is already in place. wdyt about removing this?

Reply:
Hi @laurafitzgerald,

Following the comments inline.  

> About the Prometheus Alerts Procedures, it seems to me that that is duplicating the Overview section. It's pretty much the same sentence and set of links so perhaps we could include one rather than both.

Hi @laurafitzgerald it is not. It is what makes clear what it means the critical and etc and was a request from SRE's @Nanyte25. Also, we should have a standard for that, so no make sense i just this one be different. 

> I renamed the Validate section to Check Operator so it's still in there just under a different name, the link to it references Check Operator rather than validate so it does work.

I prefer the nomenclature `Validate` which IMHO makes more clear what is the purpose of the section. Also, it came from an example provided by @Nanyte25 and I do not think that we should do things differently here. Keep the same standard is helpful for us and admins to know how to use it and have a better understanding. 

> Actually I'm wondering now if we need to include this text at all. If the user has landed on the SOP then probably the monitoring stuff is already in place. wdyt about removing this?

It allows who is checking it and has no idea over the project and how it works to get the understanding over it.